LICENSE-3rdparty.csv

```diff
@@ -1,1408 +1,1408 @@
+core,k8s.io/kubernetes/pkg/volume,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/kubernetes/pkg/volume/util/fs,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/kubernetes/pkg/volume/util/fsquota,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/kubernetes/pkg/volume/util/fsquota/common,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/kubernetes/pkg/volume/util/hostutil,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/kubernetes/pkg/volume/util/recyclerclient,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/kubernetes/pkg/volume/util/subpath,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/kubernetes/third_party/forked/golang/expansion,BSD-3-Clause,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/metrics/pkg/apis/custom_metrics,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/metrics/pkg/apis/custom_metrics/install,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/metrics/pkg/apis/custom_metrics/v1beta1,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/metrics/pkg/apis/custom_metrics/v1beta2,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/metrics/pkg/apis/external_metrics,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/metrics/pkg/apis/external_metrics/install,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/metrics/pkg/apis/external_metrics/v1beta1,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/mount-utils,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/utils/buffer,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/utils/exec,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/utils/integer,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/utils/io,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/utils/keymutex,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/utils/net,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/utils/nsenter,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/utils/path,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/utils/pointer,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,k8s.io/utils/trace,Apache-2.0,Cloud Native Computing Foundation (CNCF)
+core,sigs.k8s.io/apiserver-network-proxy/konnectivity-client/pkg/client,Apache-2.0,UNKNOWN
+core,sigs.k8s.io/apiserver-network-proxy/konnectivity-client/proto/client,Apache-2.0,UNKNOWN
+core,sigs.k8s.io/controller-runtime/pkg/scheme,Apache-2.0,UNKNOWN
+core,sigs.k8s.io/structured-merge-diff/v4/fieldpath,Apache-2.0,UNKNOWN
+core,sigs.k8s.io/structured-merge-diff/v4/merge,Apache-2.0,UNKNOWN
+core,sigs.k8s.io/structured-merge-diff/v4/schema,Apache-2.0,UNKNOWN
+core,sigs.k8s.io/structured-merge-diff/v4/typed,Apache-2.0,UNKNOWN
+core,sigs.k8s.io/structured-merge-diff/v4/value,Apache-2.0,UNKNOWN
```

Comment:
1400 - 1407, I'm pretty sure all of these can just be 'the Kubernetes Community'